setup.py

```diff
@@ -4,6 +4,37 @@
 
 from setuptools import find_packages, setup
 
+# Fail with explicit error if trying to install Django 2 on Python 2, and
+# advise on how to avoid this.
+if sys.version_info < (3,):
```

Comment:
Shouldn't this also fail on 3.0 <= Python < 3.5?

Reply:
The python_requires argument to setup() means this shouldn't happen, but I guess it is still possible!  I'll add another check and print the warning but not abort.

Reply:
As I understand `python_requires` works only with pip >= 9, it this correct? If so I think it could happen because Python 3.4 users could have pip < 9.